libs/go/libp2p_node/README.md

```diff
@@ -23,8 +23,17 @@ golangci-lint run
 staticcheck ./...
 ```
 
+For mocks generation:
+check https://github.com/golang/mock
+
 ## Messaging patterns
 
+Interaction protocol
+___
+ACN
+___
+TCP/UDP/...
+___
```

Comment:
I guess this is not the final version of the section? We could explain it textually. E.g.:

> The interactions between the node and its clients are determined by the `acn` interaction protocol (link to specification)

> ACN can potentially build on top on any transport network protocols, e.g. `TCP` or `UDP`, although the current main implementation relies on `TCP` for point-to-point communication (via `libp2p`).

Reply:
also, somewhere, link to ACN docs (when ready...)